gradle/libs.versions.toml

```diff
@@ -98,6 +99,8 @@ androidx-compose-ui-util = { module = "androidx.compose.ui:ui-util", version.ref
 # Embed XML via view binding into Composables
 androidx-compose-ui-viewBinding = { module = "androidx.compose.ui:ui-viewbinding", version.ref = "compose" }
 
+androidx-datastore-preferences = { module = "androidx.datastore:datastore-preferences", version.ref = "datastore" }
```

Comment:
@ywu96 would love your thoughts on the datastore usage here since I'm a little rusty with it